lib/main.dart

```diff
@@ -32,26 +39,44 @@ List<String> checkboxCollectionNames = [
   'PersonalPlan-Distractions',
   // Add the new table name
 ];
-Future<FirebaseApp> initializeApp() async {
+TimeOfDay calculateTime(int h, int m) {
+  TimeOfDay time = TimeOfDay(hour: h, minute: m);
+  print(time);
+  return TimeOfDay(hour: h, minute: m);
+}
+
+@pragma(
+    'vm:entry-point') // Mandatory if the App is obfuscated or using Flutter 3.1+
+void callbackDispatcher() {
+  Workmanager().executeTask((task, inputData) async {
+    int number = Random().nextInt(inputData?["text"].length);
```

Comment:
need to be more defensive here since you don't know what will be in the inputData.

Reply:
We do know because we send it(?)